src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/Licence.kt

```diff
@@ -212,4 +212,7 @@ data class Licence(
 
   @Schema(description = "The full name of the person who created licence or variation", example = "Gordon Sumner")
   val createdByFullName: String? = null,
+
+  @Schema(description = "Is this licence in PSS period?", example = "(LED < TODAY <= TUSED)")
```

Comment:
example should be `true` or `false`, but you could add that rule in the description?